plugin/yt_dlp_plugins/extractor/getpot_bgutil_http.py

```diff
@@ -27,7 +27,7 @@ class BgUtilHTTPPotProviderRH(GetPOTProvider):
     VERSION = __version__
     _SUPPORTED_PROXY_SCHEMES = (
         'http', 'https', 'socks4', 'socks4a', 'socks5', 'socks5h')
-    _SUPPORTED_FEATURES = (Features.NO_PROXY, Features.ALL_PROXY)
+    _SUPPORTED_FEATURES = (Features.NO_PROXY, Features.ALL_PROXY, ...)
```

Comment:
what's the benefit of adding the ellipsis?

Reply:
the linter doesnt like trailing commas (presumably because it looks like a mistake)

Reply:
why do we need trailing commas?
![image](https://github.com/user-attachments/assets/b4466817-3082-4d0b-b144-42ff0d7d0681)
it's an internal rh in yt-dlp

Reply:
https://discord.com/channels/807245652072857610/1272517645778026562/1285988606866882622

Reply:
if we declare support for `Features.NO_PROXY`, that is a tuple already